distribution/docs/src/main/jdocs/content/_plugins/post-process-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,17 @@
+:type: pluginIntro
+:status: published
+:title: Post-Process Plugins
+:link: _post_process_plugins
+:summary: Performs additional processing after ingest.
+:plugintypes: postprocess
+:order: 06
+
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
```

Comment:
✏️ Should we use something else than the JavaDoc comment here? Maybe something like _Post-Process plugins are experimental and, while their interface is functional and tested, they may change or be removed in the future_. Thoughts?